<?xml version="1.0"?>
<launch>

<!-- Load the robot description URDF -->
    <param name="robot_description" command="cat $(find addverb_cobot_description)/urdf/robot.urdf" />
    <!-- Load controller settings -->
    <rosparam file="$(find addverb_cobot_control)/config/default_controllers.yaml" command="load" />
    <!-- temp ros param launch -->
    <rosparam file="$(find addverb_cobot_control)/config/payload_param.yaml" command="load" />
    <!-- Load controller manager -->
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        args="joint_state_controller velocity_controller twist_controller joint_trajectory_controller effort_controller" />
    <node name="control_node" pkg="addverb_cobot_control" type="control_node" required="true" respawn="false" output="screen" />
</launch>
